lib/src/model.dart

```diff
@@ -6956,7 +6873,7 @@ class PackageBuilder {
       //                   many AnalysisDrivers
       // TODO(jcollins-g): make use of DartProject isApi()
       // TODO(keertip): Use summary 2 after fixing #2017
-      AnalysisDriver.useSummary2 = false;
+      AnalysisDriver.useSummary2 = true;
```

Comment:
Remove this?
It is `true` by default, and in general you should not access private APIs :-)

Reply:
done.